test-partner/debugpartner.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  annotations:
+    debug.openshift.io/source-container: container-00
+    openshift.io/scc: node-exporter
+  namespace: default
+  name: debug
+spec:
+  selector:
+    matchLabels:
+      test-network-function.com/app: debug
+  template:
+    metadata:
+      labels:
+        test-network-function.com/skip_connectivity_test: ""
+        test-network-function.com/app: debug
+      name: debug
+    spec:
+      nodeSelector:
+        test-network-function.com/app: debug
```

Comment:
It's a bit confusing to use the same label in pod selector and node selector. Maybe call it target instead? Similar to our pod target label.